app/views/classified_listings/index.html.erb

```diff
@@ -2,24 +2,24 @@
   <% title "Listings" %>
 
   <link rel="canonical" href="https://dev.to<%= request.path %>" />
-  <meta name="description" content="Where programmers share ideas and help each other grow.">
+  <meta name="description" content="<%= SiteConfig.community_description %>">
   <meta name="keywords" content="software development,engineering,rails,javascript,ruby">
   <meta property="og:type" content="website" />
   <meta property="og:url" content="https://dev.to<%= request.path %>" />
   <meta property="og:site_name" content="<%= community_qualified_name %>" />
   <% if @displayed_classified_listing %>
-    <meta property="og:title" content="<%= truncate @displayed_classified_listing.title, length: 54 %>" />
+    <meta property="og:title" content="<%= truncate @displayed_classified_listing.title, length: 54 %>">
```

Comment:
Looks like an additional ` />` was removed? We do this inconsistently though so not sure what's correct syntax 🙃

Reply:
Yeah we're totally inconsistent on the site and either is fine in plain old HTML. So I dunno what's right.

I removed it because that's what I thought rubocop might have been yelling at me for, but wasn't.